### ISSUE TITLE
Add support for Windows Server 2016 Family

### ISSUE THREAD

**Issue by ebjornset** [TAMUArch/cookbook.windows_ad#101]
Hi,

I've just tried to use **windows_ad_domain** to create a new AD forest om a Windows Server 2016 Core, like this:
```
windows_ad_domain "contoso.com" do
  action :create
  type "forest"
  safe_mode_pass "Passw0rd"
  domain_pass "Passw0rd"
  domain_user "Administrator"  
  options ({ "DomainMode" => "Win2012R2",
             "ForestMode" => "Win2012R2",
             "InstallDNS" => "yes",
           })
end
```

Running **kitchen coverge** fails, and here is (what I think is) the relevant output:

```    
.....
       STDOUT: The dcpromo unattended operation is replaced by the ADDSDeployment module for Windows PowerShell. For mor
e information, see http://go.microsoft.com/fwlink/?LinkId=220924

       Checking if Active Directory Domain Services binaries are installed...

       The specified argument 'DomainMode' was not recognized.
.....
```

**ohai** reports
```
 "os": "windows",
 "os_version": "10.0.14393",
 "platform": "windows",
 "platform_version": "10.0.14393",
 "platform_family": "windows",
```
on the box, so I presume the problem is caused by the comparison ```if node['os_version'] >= '6.2'``` in **providers/domain.rb** line 36 returning false?

Will it be possible to add support for Windows Server 2016, e.g. by using a numeric comparison of the first (major) version number >= 7 as well?

**Comment by derekgroh** [comment]
We don't have code to support 2016 yet.  You can make a feature request and we'll see if anyone picks it up.

**Comment by derekgroh** [comment]
Resolved with #112
